var/spack/repos/jcsda-emc-bundles/packages/ufs-weather-model-env/package.py

```diff
@@ -26,7 +26,7 @@ class UfsWeatherModelEnv(BundlePackage):
     depends_on("base-env", type="run")
     depends_on("ufs-pyenv", type="run", when="+python")
 
-    depends_on("fms@2022.04", type="run")
+    depends_on("fms@2023.01", type="run")
```

Comment:
I think you want to update that version in a few more places:
```
$ grep -Rie fms spack/var/spack/repos/jcsda-emc-bundles/*
spack/var/spack/repos/jcsda-emc-bundles/packages/ufs-srw-app-env/package.py:    depends_on("fms@2022.04")
spack/var/spack/repos/jcsda-emc-bundles/packages/ufs-weather-model-env/package.py:    depends_on("fms@2022.04", type="run")
spack/var/spack/repos/jcsda-emc-bundles/packages/jedi-ufs-env/package.py:    depends_on("fms@2022.04+fpic", type="run")
```
(note that the `fpic` version becomes `pic` in Alex's PR).

But not here (keep this version):
```
spack/var/spack/repos/jcsda-emc-bundles/packages/jedi-fv3-env/package.py:    depends_on("fms@release-jcsda", type="run")
```

Reply:
thanks for checking this, @climbfuji. changes made.

Reply:
the `ubuntu-ci-x86_64-build` ci test fails (https://github.com/JCSDA/spack-stack/actions/runs/5016762811/jobs/8994028834?pr=587) with fpic changed to pic here (probably need more changes from alex's pr). should i just keep it as fpic for this PR and associated spack-stack PR? unless alex's pr will be merged sooner. whatever works.


@AlexanderRichert-NOAA @climbfuji

Reply:
I just merged spack changes and am about to merge spack-stack changes, so that should take care of it.